src/power_grid_model/core/data_handling.py

```diff
@@ -147,14 +147,14 @@ def create_output_data(
 def process_output_component_types(
     output_type: OutputType,
     output_component_types: ComponentAttributeMapping,
-    available_components: list[ComponentType],
+    available_components: list[ComponentTypeLike],
 ) -> _ComponentAttributeMappingDict:
     """Checks valid type for output_component_types. Also checks for any invalid component names and attribute names
 
     Args:
         output_type (OutputType): the type of output that the user will see (as per the calculation options)
         output_component_types (OutputComponentNamesType):  output_component_types provided by user
-        available_components (list[ComponentType]):  all components available in model instance
+        available_components (list[ComponentTypeVar]):  all components available in model instance
```

Comment:
Given the current implementation, it should be `list[ComponentTypeLike]`, instead of `list[ComponentTypeVar]`, shouldn't it?

Reply:
yes